frontend/src/components/layout/Layout.module.scss

```diff
@@ -7,17 +7,72 @@
 
   // See https://fkhadra.github.io/react-toastify/how-to-style#override-css-variables
   --toastify-toast-width: 95%;
-  --toastify-color-success: #{$color-green-60};
-  --toastify-text-color-success: #{$color-white};
-  --toastify-color-error: #{$color-red-50};
-  --toastify-text-color-error: #{$color-white};
+  --toastify-color-light: #{$color-white};
+  --toastify-color-dark: #{$color-grey-50};
+  --toastify-color-info: #{$color-blue-50};
+  --toastify-color-success: #{$color-green-50};
+  --toastify-color-warning: #{$color-yellow-50};
+  --toastify-color-error: #{$color-red-60};
+
+  --toastify-toast-min-height: $layout-md;
+  --toastify-toast-max-height: $layout-md;
 }
 
+// Allows us to dynamically set the background color based on Toast container state
+$toastify-toast-close-button: var(--toast-close-button);
+
 .toast {
-  border-radius: $border-radius-md;
   text-align: center;
-  font-weight: 700;
+  font-weight: 400;
+  font-size: 13px;
+  line-height: 21px;
+  height: $layout-md;
+  border-radius: $border-radius-sm;
   font-family: $font-stack-base;
+  padding: 0;
+  max-width: $content-xl;
+  margin: 0 auto;
+
+  &:global {
```

Comment:
@Vinnl is this alright here? 

It allows us to target based on toast state (error, etc.) and we can update `$toastify-toast-close-button` with the correct color since they're separate but one depends on the other. Unless there is a better way.

Reply:
last question before you review @maxxcrawford - are we removing the progress bar too? I went ahead and removed it since it looks like we don't use it, based on the Figma.

Reply:
(Reply to https://github.com/mozilla/fx-private-relay/pull/1854#issuecomment-1113340516)

Ah yes, that looks fine to me, and thanks for adding the comment to the stylelintrc. Since we're using toastify to do the toasts, I don't expect us to want to re-use the global `.toast` class anywhere, so I don't expect this to cause problems down the road. Nice find!